python/kserve/requirements.txt

```diff
@@ -21,5 +21,6 @@ grpcio>=1.34.0
 tritonclient==2.18.0
 protobuf~=3.19.0
 prometheus-client>=0.13.1
+portforward
```

Comment:
this looks like a bit risky dependency, can we only install it in the e2e test?

Reply:
yes was testing it

Reply:
moved